doc/man7/store.pod

```diff
@@ -0,0 +1,104 @@
+=pod
+
+=head1 NAME
+
+store - Store retrieval functions
```

Comment:
Probably all the above library man pages should reference this page somewhere prominent near the beginning of the page (and in the SEE ALSO section).